src/hotspot/cpu/aarch64/aarch64_sve.ad

```diff
@@ -1,6 +1,6 @@
 //
-// Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
-// Copyright (c) 2020, Arm Limited. All rights reserved.
+// Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Usually the year of introduction is preserved, so this should be `2020, 2021,`

Reply:
Thanks, updated